lib/data/transport.js

```diff
@@ -48,7 +48,11 @@ class DataTransport extends EventEmitter {
    */
   _publish(message) {
     const data = JSON.stringify(message);
-    this._dataChannel.send(data);
+    try {
+      this._dataChannel.send(data);
+    } catch (error) {
+      // Do nothing.
```

Comment:
same. when is this expected - how about logging the errors?

Reply:
@makarandp0 In integration tests, sometime when you disconnect from a Room, if the timing is right, trying to send data on RTCDataChannels whose RTCPeerConnection is closed throws an error. This makes sure that the tests don't fail because of it. Regarding logging, we should do it once we thread the logging module down to this level.